<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Κρατήσεις</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" alignment="Qt::AlignRight">
     <widget class="QComboBox" name="monthSelection">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maxVisibleItems">
       <number>12</number>
      </property>
      <item>
       <property name="text">
        <string>Ιανουάριος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Φεβρουάριος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Μάρτιος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Απρίλιος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Μάιος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ιούνιος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ιούλιος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Αύγουστος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Σεπτέμβριος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Οκτώβριος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Νοέμβριος</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Δεκέμβριος</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
